app/services/NextUpdatesService.scala

```diff
@@ -16,29 +16,30 @@
 
 package services
 
+import java.time.LocalDate
+
 import auth.MtdItUser
 import connectors._
+import javax.inject.{Inject, Singleton}
 import models.incomeSourceDetails.IncomeSourceDetailsModel
-import models.reportDeadlines._
+import models.nextUpdates._
 import play.api.Logger
 import uk.gov.hmrc.http.{HeaderCarrier, InternalServerException}
 
-import java.time.LocalDate
-import javax.inject.{Inject, Singleton}
```

Comment:
Just a note: probably imports were optimised in some older version of Intellij IDEA or its Scala plugin. The newer versions put standard java/scala imports from top to here, the bottom of imports section :)